x-pack/dev-tools/api_debug/request_from_api.js

```diff
@@ -9,6 +9,7 @@ import fetch from 'node-fetch';
 import { resolve } from 'path';
 import abab from 'abab';
 import pkg from '../../package.json';
+import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common/src/constants';
```

Comment:
@elastic/kibana-operations, do you own `dev-tools`? It's missing from the `CODEOWNERS` file.

Reply:
We're probably the closest team, although I don't think any of it's used anymore.  Feel free to add us or we can folow up.